index.bs

```diff
@@ -2439,11 +2439,9 @@ private key=] are defined in [[#terminology]].
 The [=authentication ceremony structure=] is defined by the [=authentication factors=] used in the [=authentication
 ceremony|ceremony=]. Different [=authentication ceremony structures=] provide different benefits for security and usability.
 
-The following discussion assumes that the [=[RP]=] can verify the security guarantees of the [=authenticator=] by inspecting
-[=attestation statements=] generated by the [=authenticator=]. In particular, the [=[RP]=] cannot trust that the [=authenticator=]
-can verify [=something you know=] or [=something you are=] unless these properties can be derived from the [=attestation
-statement=]. Thus, an [=authenticator=] that cannot provide such an [=attestation statement=] can only enable the [=second-factor
-use case=]. See also [[#sctn-no-attestation-security-attestation]].
+The following discussion assumes that the [=[RP]=] to some degree trusts the [=authenticator=] to perform [=user verification=].
```

Comment:
suggest something like:

The following discussion is based on the notion that the degree of trust an [=[RP]=] has in an [=authenticator=] to satisfactorily perform [=user verification=] is established and assessed by the [=[RP]=].

Reply:
Looks good, thanks!